<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TvShowSearch</class>
 <widget class="QDialog" name="TvShowSearch">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>714</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Search Result</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../MediaElch.qrc">
    <normaloff>:/img/MediaElch.png</normaloff>:/img/MediaElch.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#TvShowSearch {
	background-image: url(:/img/pattern_dark.png);
}</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="MyLineEdit" name="searchString"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTableWidget" name="results">
       <property name="styleSheet">
        <string notr="true">alternate-background-color: rgb(243, 246, 250);</string>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="showGrid">
        <bool>false</bool>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string>Result</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="styleSheet">
        <string notr="true">QGroupBox {
	border: 1px solid rgba(0, 0, 0, 100);
	border-radius: 5px;
	background-color: rgba(255, 255, 255, 200);
}</string>
       </property>
       <property name="title">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Info to load</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>8</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="MyCheckBox" name="chkTitle">
          <property name="text">
           <string>Title</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="MyCheckBox" name="chkRating">
          <property name="text">
           <string>Rating</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="MyCheckBox" name="chkFirstAired">
          <property name="text">
           <string>First aired</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="MyCheckBox" name="chkDirector">
          <property name="text">
           <string>Director</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="MyCheckBox" name="chkWriter">
          <property name="text">
           <string>Writer</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="MyCheckBox" name="chkCertification">
          <property name="text">
           <string>Certification</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="MyCheckBox" name="chkOverview">
          <property name="text">
           <string>Overview</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="MyCheckBox" name="chkGenres">
          <property name="text">
           <string>Genres</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="MyCheckBox" name="chkNetwork">
          <property name="text">
           <string>Network</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="MyCheckBox" name="chkSeasonEpisode">
          <property name="toolTip">
           <string>Logo, Clear Art, CD Art</string>
          </property>
          <property name="text">
           <string>Season/Episode</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="MyCheckBox" name="chkActors">
          <property name="text">
           <string>Actors</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="MyCheckBox" name="chkFanart">
          <property name="text">
           <string>Fanart</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="MyCheckBox" name="chkPoster">
          <property name="text">
           <string>Poster</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="MyCheckBox" name="chkSeasonPoster">
          <property name="text">
           <string>Season Poster</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="MyCheckBox" name="chkSeasonBackdrop">
          <property name="text">
           <string>Season Fanart</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="MyCheckBox" name="chkSeasonBanner">
          <property name="text">
           <string>Season Banner</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="MyCheckBox" name="chkBanner">
          <property name="text">
           <string>Banner</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="MyCheckBox" name="chkThumbnail">
          <property name="text">
           <string>Thumbnail</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="MyCheckBox" name="chkExtraArts">
          <property name="text">
           <string>Artwork</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="chkUnCheckAll">
          <property name="text">
           <string>(Un)Check all</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QComboBox" name="comboUpdate">
       <item>
        <property name="text">
         <string>Update TV Show only</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Update TV Show and new Episodes</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Update TV Show and all Episodes</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Update new Episodes</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Update all episodes</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonClose">
       <property name="text">
        <string>Close</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyCheckBox</class>
   <extends>QCheckBox</extends>
   <header location="global">smallWidgets/MyCheckBox.h</header>
  </customwidget>
  <customwidget>
   <class>MyLineEdit</class>
   <extends>QLineEdit</extends>
   <header>smallWidgets/MyLineEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../MediaElch.qrc"/>
 </resources>
 <connections/>
</ui>
